packages/eslint-config-react/package.json

```diff
@@ -15,6 +15,7 @@
     "typescript": ">=4.2"
   },
   "dependencies": {
+    "@trampoline/eslint-config": "^1.0.0-alpha.0",
```

Comment:
You should be using version `workspace:*`, which will use the version in the current workspace instead. Otherwise, we'll need to update the version number here as well when we release the packages 😉

Reply:
in this case do we omit the `@trampoline` part of the package name? so it would be `"eslint-config": "workspace:*"`